scv-prototype-frontend/src/app/components/Button/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+
+	const { type,
+		variant,
+		size,
+		disabled,
+		children,
+		className,
+		...otherProps } = props;
+
+	const disabledProp = disabled ? { disabled: "disabled" } : undefined;
+
+	return <button type={type}
+		className={
+			"btn" +
+			` btn-${variant}` +
+			(size && size !== Button.sizes.default ? ` btn-${size}` : "") +
+			(className ? ` ${className}` : "")
+		}
+		{...disabledProp}
```

Comment:
Is this necessary? React is usually pretty smart when it comes to this stuff. I mean, you can use `disabled={disabled}` and if the value is undefined, it won't render it. (**I think**)

Reply:
I was not sure how to implement it. I will try your approach.